CONTRIBUTING.md

```diff
@@ -2,45 +2,69 @@
 
 ## Getting started
 
-### Installing
+### Machine setup
 
-Once the repo has been cloned, install Lerna:
+To work with the FAST-DNA [monorepo](https://en.wikipedia.org/wiki/Monorepo) you'll need Git, Node.js, Yarn, and Lerna setup on your machine.
+
+FAST-DNA uses Git as its source control system. If you haven't already installed it, you can download it [here](https://git-scm.com/downloads) or if you prefer a GUI-based approach, try [GitHub Desktop](https://desktop.github.com/).
+
+Once Git is installed, you'll also need Node.js, which FAST-DNA uses as its JavaScript runtime, enabling its build and test scripts. Node.js instructions and downloads for your preferred OS can be found [here](https://nodejs.org/en/).
+
+Because the FAST-DNA repository is structured as a monorepo, we'll need a couple of tools to manage that. The first is Yarn, which can be installed by executing the following command at the terminal:
+
+```shell
+npm install -g yarn
+```
+
+The second tool you'll need is Lerna, which can be installed with this command:
 
 ```bash
 yarn global add lerna
 ```
 
-Install all package dependencies and build all workspaces (local dependencies):
+:::important
+The above steps are a one-time setup for your machine and do not need to be repeated after the initial configuration.
+:::
```

Comment:
I'm not sure if this was supposed to be formatted a certain way, but all that's coming up for me is the ::: in github, does docusaurus do something with that?

Reply:
Yes, it will display what they call an "admonition" which is basically a card style with a particular color to highlight this as important. In GitHub it will just format as normal text though.